examples/monaco-editor-webpack-plugin/webpack.config.js

```diff
@@ -38,13 +32,14 @@ export default {
       customLanguages: [
         {
           label: 'yaml',
+          // eslint-disable-next-line prettier/prettier
           entry: [
-            'monaco-yaml/lib/esm/monaco.contribution',
+            'monaco-yaml/index',
             'vs/basic-languages/yaml/yaml.contribution',
```

Comment:
Does it work if you change this to just `'monaco-yaml'`?

Reply:
Yeah, it does :)